libs/stache/src/lib/modules/nav/link.directive.ts

```diff
@@ -14,50 +14,55 @@ import { StacheRouteService } from '../router/route.service';
 
 import { StacheNavService } from './nav.service';
 
-// eslint-disable-next-line @angular-eslint/no-input-rename
 @Directive({
   selector: '[stacheRouterLink]',
 })
 export class StacheRouterLinkDirective implements OnChanges, AfterViewInit {
-  private _stacheRouterLink = '';
-
-  // eslint-disable-next-line @angular-eslint/no-input-rename
-  @Input('stacheRouterLink')
-  set stacheRouterLink(routerLink: string) {
-    if (routerLink === '.') {
-      this._stacheRouterLink = this.routerService.getActiveUrl();
+  @Input()
+  public set stacheRouterLink(value: string | string[] | undefined) {
```

Comment:
I had to add `string[]` as a possible type because the layout components were passing arrays to this input. Strict-mode caught this.